Products/ZenUI3/browser/resources/js/zenoss/date.js

```diff
@@ -372,4 +341,102 @@ Ext.define('Zenoss.grid.column.DateTime', {
     }
 });
 
+
+/**
+ * @class Zenoss.DateRange
+ * @extends Ext.form.field.Date
+ * A DateRange
+ */
+Ext.define("Zenoss.DateRange", {
+    extend: "Ext.form.field.Date",
+    alias: ['widget.DateRange'],
+    xtype: "daterange",
+    trigger1Cls : 'x-form-date-trigger',
+    trigger2Cls : 'x-form-arrow-trigger',
```

Comment:
this way we add new trigger into datefield, than we can catch trigger click in `onTrigger2Click` fn.